### ISSUE TITLE
After entering the text and selecting another tool, flameshot closes

### ISSUE THREAD

**Issue by xaozai** [flameshot-org/flameshot#2434]
### Flameshot Version

 flameshot-11.0.0-1.debian-11.amd64

### Installation Type

Linux, MacOS, or Windows Package manager (apt, pacman, eopkg, choco, brew, ...)

### Operating System type and version

Linuxmint 20.3 una; Linux 5.4.0-99-generic

### Description

After entering the text in the screenshot and selecting another tool, flameshot closes.

### Steps to reproduce

1. Run 'flameshot gui'
2. Select a region on the primary monitor
3. Select a text input tool
4. Enter some text
5. Choose another tool, such as a pencil.

Immediately after step 5 the screenshot editing window disappears.

### System Information

1. Linuxmint 20.3 una; Linux 5.4.0-99-generic; x64
2. Notebook; 1600 x 900; 60.1 Hz
3. Xfce 4.16; Xorg

**Comment by mmahmoudian** [comment]
Isn't this duplicate of all the following?
- #2283
- #2353
- #2402
- #2407
- #2429

**Comment by xaozai** [comment]
Yes, it looks like it's all about the same thing. I'm sorry.

**Comment by mmahmoudian** [comment]
@xaozai No problem. Thanks for taking the time to help us improve the software.
